### PR TITLE
Fix problem with upgrade and missing provide

### PR DESCRIPTION
**Pull request by j-mracek**
https://bugzilla.redhat.com/show_bug.cgi?id=1263888#c83

### CODE REVIEW

File: dnf-plugins-extras.spec [hunk 1]
```diff
@@ -32,6 +32,7 @@ Extras Plugins for DNF.
 
 %package -n %{name}-common-data
 Summary:        Common data files for Extras Plugins for DNF
+Provides:       %{name} = %{version}-%{release}
```
Comment (ignatenkobrain) [inline dnf-plugins-extras.spec]: that doesn't make any sense. when people do `dnf install dnf-plugins-extras`, it probably should do something else than installing man pages ;)

Comment (j-mracek) [inline dnf-plugins-extras.spec]: You mean revert this commit: https://github.com/rpm-software-management/dnf-plugins-extras/commit/2ae5079678c3711b50cea54d4fc5848e8ba0618e

Comment (ignatenkobrain) [inline dnf-plugins-extras.spec]: yeah, to some degree. probably we should revert with requirements ONLY for packages which do not require anything else (I think candidates are repomanage and debug)....

**Comment by jsilhan** [comment]
FTR from today's meeting this change was rejected. We don't want to introduce `dnf-plugins-extras` package again because of previous issue with the packaging. Components should require exact plugins to not pull additional dependencies.

**Pull request closed without merging**